lib/travis/build/script/go.rb

```diff
@@ -103,6 +108,13 @@ def uses_make
             '-f GNUmakefile || -f makefile || -f Makefile || -f BSDmakefile'
           end
 
+	  def uses_15_vendoring
+            if (go_version < '1.5' || go_version == 'go1')
+              'false'
+            end
+	    (go_version < '1.6') ? '$GO15VENDOREXPERIMENT == 1' : '$GO15VENDOREXPERIMENT != 0'
```

Comment:
Good idea with the docstring. `Gem::Version` was discussed above, so I'll give that a shot and see if I can't make it work. I'm not actually a Rubyist, but this was straightforward enough I took a crack at it anyways.